src/pages/results/results.page.tsx

```diff
@@ -1,12 +1,77 @@
-import React from 'react';
-import './results.scss';
+import React, { useEffect, useState } from 'react';
+import { Select } from 'antd';
+import Abstract from "../../components/abstract/abstract";
 
-interface IProps { }
 
-const Results = (props: IProps) => {
-    return (
-        <div className='results'>Results</div>
-    )
-}
+const { Option } = Select;
+const [loading] = useState(false);
+
+
+TODO: once the back-end is ready this comment code can be used to fetch the whole tages from the database.
+// const Results = () => {
+//   const [options, setOptions] = useState([]);
+//   const [loading, setLoading] = useState(true);
+//   const [searchArray, setSearchArray] = useState([]);
+
+//   useEffect(() => {
+//     fetch('/tags')
+//       .then(response => response.json())
+//       .then(data => {
+//         setOptions(data);
+//         setLoading(false);
+//       });
+//   }, []);
+
+
+const Results = () => {
+  const [options] = useState([
```

Comment:
Why do you have a `useState` hook with no `setState` function extracted? explain your reasoning.

Reply:
its just for testing now, and this code will be replace with the request above of it. however I did this also